docs/contributors/outreach.md

```diff
@@ -35,7 +35,7 @@ A short list of useful articles around defining, extending, and contributing to
 
 ## Talks
 
-Talks given about Gutenberg, including slides and videos as they are available.
+Talks were given about Gutenberg, including slides and videos as they are available.
```

Comment:
This changes the meaning of this description and is incorrect IMHO

Reply:
Thanks :)